definitions/pipelines/somatic_exome.cwl

```diff
@@ -137,6 +137,16 @@ inputs:
         default: true
     manta_output_contigs:
         type: boolean?
+    vcf:
```

Comment:
This is too generic of a name for an input to add to a top-level pipeline.  There are many steps in this workflow and other VCFs are already inputs. It should be made clear what the VCF is.